python/plugins/processing/algs/gdal/gdalcalc.py

```diff
@@ -105,24 +172,32 @@ def group(self):
     def groupId(self):
         return 'rastermiscellaneous'
 
+    def commandName(self):
+        if isWindows():
```

Comment:
commandName doesn't include the .py extension - this should always return 'gdal_calc'